app/templates/serviceLearning/slcProposal.html

```diff
@@ -198,3 +205,7 @@ <h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
   </div>
   <input value={{course.id}} name="courseID" hidden/>
 </div>
+
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
```

Comment:
You definitely didn't need these just to show the status. Where did they come from?

Reply:
Removed.